packages/react/src/ActionList/Divider.tsx

```diff
@@ -1,24 +1,14 @@
 import type React from 'react'
-import type {SxProp} from '../sx'
 import {clsx} from 'clsx'
 import classes from './ActionList.module.css'
-import {BoxWithFallback} from '../internal/components/BoxWithFallback'
 
-export type ActionListDividerProps = SxProp & {
+export type ActionListDividerProps = {
   className?: string
 }
 
 /**
- * Visually separates `Item`s or `Group`s in an `ActionList`.
+ * Visually separates `Items or `Groups in an `ActionList`.
  */
-export const Divider: React.FC<React.PropsWithChildren<ActionListDividerProps>> = ({sx, className}) => {
-  return (
-    <BoxWithFallback
-      className={clsx(className, classes.Divider)}
-      as="li"
-      aria-hidden="true"
-      sx={sx}
-      data-component="ActionList.Divider"
-    />
-  )
+export const Divider: React.FC<React.PropsWithChildren<ActionListDividerProps>> = ({className}) => {
+  return <li className={clsx(className, classes.Divider)} aria-hidden="true" data-component="ActionList.Divider" />
```

Comment:
let's make sure we support style here, all components should support style in the absence of sx 🙏